src/Schema/Events/Inquiries.ts

```diff
@@ -0,0 +1,46 @@
+import { ActionType } from "../Event"
+
+/**
+ * Schemas describing inquiry events
+ * @packageDocumentation
+ */
+
+/**
+ * A user successfully sends an artwork inquiry request to a partner.
+ *
+ * This schema describes events sent to Segment from [[sentArtworkInquiry]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "sentArtworkInquiry",
+ *    artwork_id: "582b1bc09c18db3b0c000999",
+ *    artwork_slug: "philippe-halsman-alfred-hitchcock-during-the-filming-of-the-birds",
+ *    inquiry_id: "614e3063ef0b81000bc4f45a",
+ *    products: [
+        {
+          price: 6000,
```

Comment:
I'm pretty sure this is a pre-formatted string most of the time? The default just happens to be 6000 for some reason?